### DISCUSSION TITLE
Additional Resource

### DISCUSSION THREAD

**Discussion by morydd** [BrenekH/yamaha-rcp-docs#1] [General]
We've been working on finding similar information, but it's great to see someone putting it together. I'd be happy to help with this project. 
Using information from [https://usa.yamaha.com/files/download/other_assets/9/1258239/190118_mtx_mrx_xmv_ex_rcps_v331_rev13_en.pdf](url) we've tested the `getn` `setn` and `setr` commands, and they seem to work as well.

**Comment by BrenekH** [comment]
Sweet! Thanks for the link, I'll add it to the README and spend some time going through it myself.

As an aside, when you say you've tested the `getn`, `setn`, and `setr` commands, which device was that on?

**Reply by morydd** [reply]
I tested them on our TF5.

**Comment by snurrgnu** [comment]
Just wanted to add a link to more Yamaha commands.
The OSC-commands for the Rivage series mixers
https://uk.yamaha.com/files/download/other_assets/5/1407565/RIVAGE_PM_osc_specs_v102_en.pdf
